org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/thingTranslator.js

```diff
@@ -214,6 +218,110 @@ class Translator {
                 return response.targetTemperatureLow.temperature;
             });
     }
+
+    subscribe(options) {
+        return this.commonsubscribe(deviceType, deviceId, options.callbackUrl, options.secret);
+    }
+
+    unsubscribe(subscriptionId) {
+        return this.commonunsubscribe(deviceType, deviceId, subscriptionId);
+    }
+
+    getSubscriptions() {
+        // GET /sensor_pods/<sensor pod id>/subscriptions
+        var requestUri = 'https://api.wink.com/' + deviceType + '/' + deviceId + '/subscriptions';
+
+        // Set the headers
+        var headers = {
+            'Authorization': 'Bearer ' + accessToken
+        }
+
+        // Configure the request
+        var options = {
+            url: requestUri,
+            method: 'GET',
+            headers: headers,
+            followAllRedirects: true
+        }
+
+        // Start the async request
+        return request(options)
+            .then(function (body) {
+                // request succeeded.
+                return JSON.parse(body);
+            })
+            .catch(function(reason) {
+                console.log('Error: ' + reason);
+            });
+    }
+
+    // Subscribe to Wink notifications
+    // serviceurl - The url endpoint set up to receive postbacks and manage verification
+    // secret - Subscriber generated secret for HMAC computation (if omitted, HMAC digest will
+    //      not be present on callbacks)
+    // This method passes the PubSubHubbub a subscriber URL to the Wink device so that the subscriber
+    // will receive postbacks.  This subscription needs to be refreshed or it will expire (currently 24 hrs).
+    commonsubscribe(deviceType, deviceId, serviceUrl, secret) {
```

Comment:
>commonsubscribe [](start = 4, length = 15)

as you note below, my expectation is that both common *methods should go in the wink hub